### DISCUSSION TITLE
OpenMPI in DeepXDE

### DISCUSSION THREAD

**Discussion by Diganta5** [lululxvi/deepxde#1198] [Q&A] [answered]
Is it possible to use OpenMPI with DeepXDE for domain decomposition?

**Comment by lululxvi** [comment, marked as answer]
No. you need to implement it by yourself.

**Reply by Diganta5** [reply]
I have tried to implement open mpi(mpi4py). When I am using 2 processes it is working well but when I am using more than 2 it is not working.

How I implemented:
I am saving a model in a folder and sending the epoch number to the target process.

It will be better if you suggest how to implement it.

**Reply by lululxvi** [reply]
See ongoing effort at https://github.com/lululxvi/deepxde/pull/1094, https://github.com/lululxvi/deepxde/pull/1205
